Assets/Scripts/BuffPanelSDK/BuffPanel.cs

```diff
@@ -11,8 +11,9 @@ class BuffPanelMonoBehaviour : MonoBehaviour
 
     public class BuffPanel
     {
-        public static string serviceHostname = "buffpanel.com";
+        public static string serviceHostname = "staging.buffpanel.com:8080";
         public static string servicePath = "/api/run";
+        public static string redirectURI = ".redirect.staging";
```

Comment:
From what i understand it should be enough to change this to '.trbt.it'
However I am surprised you aren't using the full hostname, i.e. '.redirect.staging.buffpanel.com', if there is some strange string concatenation at play, please make sure it works as intended